src/Altinn.Platform/Altinn.Platform.Events/Functions/Startup.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.Azure.WebJobs.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+Microsoft.Extensions.DependencyInjection
```

Comment:
missing using keyword and otherwise duplicate of the line above?
would think this should have given a build error?

Reply:
Oops, didn't mean to change that, as this is the function from Altinn.Platform.Events. Would also expect the PR-pipeline for functions to fire here, and fail..